engine/source/output/h3d/h3d_results/h3d_quad_scalar.F

```diff
@@ -86,7 +87,7 @@ SUBROUTINE H3D_QUAD_SCALAR(
      .   IGEO(NPROPGI,NUMGEO),INVERT(*), NV46,ID_ELEM(*),
      .   INFO1,INFO2,IS_WRITTEN_QUAD(*),IPARTQ(*),IPARTTG(*),H3D_PART(*),
      .   LAYER_INPUT ,IPT_INPUT,GAUSS_INPUT,PLY_INPUT,IUVAR_INPUT,
-     .   IR_INPUT,IS_INPUT,IT_INPUT
+     .   IR_INPUT,IS_INPUT,IT_INPUT,ID
```

Comment:
INTENT statement is missing

Reply:
Hi @sebastienVilleneuve 

The missing INTENT attributes have been added :-) ! Thanks a lot.